plugins/modules/system/capabilities.py

```diff
@@ -110,10 +110,16 @@ def getcap(self, path):
         #   '/foo'
         # If the file does not exist the output will be (with rc == 0...):
         #   '/foo (No such file or directory)'
-        if rc != 0 or (stdout.strip() != path and stdout.count(' =') != 1):
+        if rc != 0 or stderr != "":
             self.module.fail_json(msg="Unable to get capabilities of %s" % path, stdout=stdout.strip(), stderr=stderr)
         if stdout.strip() != path:
-            caps = stdout.split(' =')[1].strip().split()
+            if stdout.count(' ='):
```

Comment:
The following is more efficient (and IMO easier to understand):
```suggestion
            if ' =' in stdout:
```

Reply:
thanks